# HG changeset patch
# User Ben Abbott <bpabbott@mac.com>
# Date 1290043030 18000
#      Wed Nov 17 20:17:10 2010 -0500
# Node ID 79b77d71d01e6936ee7c2dbc2bf70abf55f7fe83
# Parent  c78247b664fc7f11422b0e0104377e843c8fce9f
subsindex.m: Fix typo in doc-string.

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,12 @@
+2010-11-17  Ben Abbott <bpabbott@mac.com>
+
+	* general/subsindex.m: Fix typo in doc-string.
+
 2010-11-17  John W. Eaton  <jwe@octave.org>
 
 	* strings/strchr.m: Style fixes.
 
 2010-11-17  Kim Hansen  <kim@ange.dk>
 
 	* strings/strchr.m: New tests.
 
diff --git a/scripts/general/subsindex.m b/scripts/general/subsindex.m
--- a/scripts/general/subsindex.m
+++ b/scripts/general/subsindex.m
@@ -23,17 +23,17 @@
 ## overloading method that allows the conversion of this class object to
 ## a valid indexing vector.  It is important to note that
 ## @code{subsindex} must return a zero-based real integer vector of the
 ## class "double".  For example, if the class constructor
 ##
 ## @example
 ## @group
 ## function b = myclass (a)
-##  b = myclass (struct ("a", a), "myclass");
+##  b = class (struct ("a", a), "myclass");
 ## endfunction
 ## @end group
 ## @end example
 ##
 ## @noindent
 ## then the @code{subsindex} function
 ##
 ## @example
@@ -57,8 +57,9 @@
 ## @end example
 ##
 ## @seealso{class, subsref, subsasgn}
 ## @end deftypefn
 
 function idx = subsindex (a)
   error ("subsindex: not defined for class \"%s\"", class(a));
 endfunction
+
